todo_app/data/trello_items.py

```diff
@@ -0,0 +1,106 @@
+import requests
+from todo_app.data.todo_item import TodoItem
+from todo_app.trello_config import TrelloStaticDataConfig
+
+class TrelloManager:
+    def __init__(self):
+        self.base_url = TrelloStaticDataConfig.BASE_URL
+        self.board_id = TrelloStaticDataConfig.BOARD_ID
+        self.api_key = TrelloStaticDataConfig.TRELLO_API_KEY
+        self.api_token = TrelloStaticDataConfig.TRELLO_TOKEN
+        
+    def create_card(self, name, desc):
+        """
+        Create a new Trello card.
+        
+        Args:
+          name: The name of the item.
+          desc: The description of the item.
+        """
+        params = {
+            'idList' : TrelloStaticDataConfig.TRELLO_TO_DO,
+            'name' : name,
+            'desc': desc,
+            "key": self.api_key,
+            "token": self.api_token
+        }
+      
+        new_card_endpoint = "cards"
+        response = requests.post(self.base_url + new_card_endpoint, params=params)
+        if response.status_code == 200:
+            print(f"Created card with name: {name}")
+        else:
+            print(f"Failed to create card with name: {name}, Status code: {response.status_code}")
```

Comment:
Really minor, but requests provide a `.ok` property on their responses to check for happy status codes, so if for example this started to return a 201 - Accepted or 204 - No Content then it would still be happy:
```suggestion
        if response.ok:
            print(f"Created card with name: {name}")
        else:
            print(f"Failed to create card with name: {name}, Status code: {response.status_code}")
```

Reply:
Another interesting point here - is this logging a good idea? We'll talk about it more later,  but we usually want to avoid any personal data ending up in our logs, and this feels ripe for that accidentally happening - imagine if users created to-dos like:
* Send private message to <email>
* Phone doctor about <condition>
* Change all my passwords from "Chelsea123"

Generally, sticking to printing out ids is a good way to avoid this